dcspkg/src/lib.rs

```diff
@@ -1,5 +1,12 @@
 use serde::{Deserialize, Serialize};
 
+mod commands;
+pub mod config;
+pub mod util;
+
+pub use crate::commands::{list_all_packages, install_package, run_package};
+pub use crate::util::list_installed_packages;
```

Comment:
You can remove this line: the function is exported anyway as util is a `pub mod`: see the site generated by `cargo doc` and it's there twice.